cmd/podman/containers/cp.go

```diff
@@ -50,12 +50,14 @@ var (
 
 var (
 	cpOpts entities.ContainerCpOptions
+	chown  bool
 )
 
 func cpFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
 	flags.BoolVar(&cpOpts.Extract, "extract", false, "Deprecated...")
 	flags.BoolVar(&cpOpts.Pause, "pause", true, "Deprecated")
+	flags.BoolVarP(&chown, "archive", "a", false, "Archive mode (copy all uid/gid information)")
```

Comment:
On `docker` this is by default false so no `chown` is done. Since `podman` should be drop in replacement for `docker` I made it false by default here, however this might be surprising for current users of `podman`.

Reply:
@vrothberg shall I change the default value to `true` to make it backward compatible with older `podman`?

Reply:
Thanks for checking. I think we should remain backwards compat with Podman, so `true` may be a good way forward.

Other may feel differently though. @rhatdan @mheon WDYT?

Reply:
I made it true by default